packages/camera/camera/example/lib/main.dart

```diff
@@ -348,11 +348,8 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
     if (controller != null) {
       await controller.dispose();
     }
-    controller = CameraController(
-      cameraDescription,
-      ResolutionPreset.medium,
-      enableAudio: enableAudio,
-    );
+    controller = CameraController(cameraDescription, ResolutionPreset.medium,
+        enableAudio: enableAudio, imageFormatGroup: ImageFormatGroup.jpeg);
```

Comment:
nit: Add a comma after the last parameter and the formatting will be nicely underneath each other:

```dart
    controller = CameraController(
      cameraDescription,
      ResolutionPreset.medium,
      enableAudio: enableAudio,
      imageFormatGroup: ImageFormatGroup.jpeg,
    );
```